From 50b2d1bb0f6455db61391ef94b813e53bf0d6986 Mon Sep 17 00:00:00 2001
From: Zhang Zhijie <zhangzj@rock-chips.com>
Date: Tue, 16 Jan 2018 09:06:02 +0800
Subject: [PATCH] arm64: dts: rockchip: px30: add firmware and optee node

Add optee node to supply OP-TEE required properties.
/optee node is supposed to be below /firmware node.

Change-Id: I0737e7ef3cb495d1fae6354baf66c600fb02fc2f
Signed-off-by: Zhang Zhijie <zhangzj@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/px30.dtsi b/arch/arm64/boot/dts/rockchip/px30.dtsi
index fcbd165eace4..60f2024a0fd3 100644
--- a/arch/arm64/boot/dts/rockchip/px30.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30.dtsi
@@ -76,6 +76,13 @@
 		status = "disabled";
 	};
 
+	firmware {
+		optee {
+			compatible = "linaro,optee-tz";
+			method = "smc";
+		};
+	};
+
 	psci {
 		compatible = "arm,psci-1.0";
 		method = "smc";
-- 
2.35.3

